File: Backend/bagverse-backend/src/main/java/com/bagverse/bagverse_backend/service/CartService.java
package com.bagverse.bagverse_backend.service;

import com.bagverse.bagverse_backend.entity.Cart;
import java.util.List;

public interface CartService {

    Cart addToCart(Cart cart);

    List<Cart> getCartByUserId(Long userId);

    void removeFromCart(Long id);
}

File: FrontEnd/bagverse-frontend/src/services/CartService.js
import API from "./api";

export const addToCart = (data) => {
  return API.post("/cart/add", data);
};

export const getCartByUser = (userId) => {
  return API.get(`/cart/user/${userId}`);
};

export const removeFromCart = (id) => {
  return API.delete(`/cart/${id}`);
};
